index.html

```diff
@@ -131,6 +131,22 @@ <h2>Inclusivity &amp; Diversity</h2>
         The application phase has ended and applicants will be contacted.
       </section>
 
+      <section id="sponsor-bronze">
```

Comment:
question: why did you put the section on a random spot? I'd suggest to put it under the liver sponsor list to have one block of sponsors

Reply:
Similar to the last year's we want to add them between the content elements.

This is why they are not at one central place

Reply:
Then we may should rename the menu link into `Sponsoring` since it's a bit misleading because it's not not about (the list of) sponsors, but about we're looking for sponsors

Reply:
Good point.
I changed it to `Sponsoring`